README.md

```diff
@@ -51,6 +51,48 @@ GENESYSCLOUD_REGION
 
 For any issues, questions, or suggestions for the provider, visit the [Genesys Cloud Developer Forum](https://developer.mypurecloud.com/forum/)
 
+### Proxy Configuration
+
+Using of a proxy is accomplished by setting the proxy settings for the provider
+
+The `Proxy` has 3 properties that determine the URL for proxying.
```

Comment:
You should add another space between this line and the next because markdown isn't picking them up properly. I checked it by opening this file in vs code on my machine and holding `SHIFT+COMMAND+V`